### DISCUSSION TITLE
Filamentphp in Shared Hosting

### DISCUSSION THREAD

**Discussion by NazarAli** [filamentphp/filament#15087] [Help]
i have problem when upload my project in shared hosting i not  have sub domain to add the request to public dir
and when try to access the project by write the public in url this error message show

The POST method is not supported for route admin/login

i used filament 3 with laravel 10 and php 8.2.26

**Comment by leandrocfe** [comment]
This isn’t a Filament issue, though. Maybe you could look up articles on how to host a Laravel project on shared hosting.

**Comment by amimhayden22** [comment]
Is it possible? If not, try making sure `APP_URL` in `.env` is set and try running the following command `php artisan route:clear`

**Reply by tahinuralam** [reply]
Delete the config file

Bootstrap/cache/config.php
